m2cgen/interpreters/r/interpreter.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from m2cgen import ast
+from m2cgen.interpreters import mixins, utils
+from m2cgen.interpreters.interpreter import ToCodeInterpreter
+from m2cgen.interpreters.r.code_generator import RCodeGenerator
+
+
+class RInterpreter(ToCodeInterpreter,
+                   mixins.LinearAlgebraMixin,
+                   mixins.BinExpressionDepthTrackingMixin,
+                   mixins.SubroutinesAsFunctionsMixin):
+
+    bin_depth_threshold = 25
```

Comment:
Just a heuristic to pass tests with the hope that `50 / 2` will be enough for the most use cases.
Refer to https://github.com/BayesWitnesses/m2cgen/pull/139#issuecomment-569084447.
Asking users to re-compile R from sources is not a good idea I guess 😃 : 
> It is hardcoded (as 50) in 6 places in gram.c, so fairly easily changed. 
> https://r.789695.n4.nabble.com/contextstack-overflow-tp818869p818870.html

Reply:
Yeah, if you can leave a comment with some reasoning behind this number - that'd be fantastic.